composer.json

```diff
@@ -10,6 +10,11 @@
 			"name": "Stefan Froemken",
 			"email": "projects@jweiland.net",
 			"role": "Developer"
+		},
+		{
+			"name": "Hoja Mustaffa Abdul Latheef",
+			"email": "hlatheef@jweiland.net",
```

Comment:
We personally will never give support. That's why we should use the "projects" mail address